src/charm.py

```diff
@@ -43,12 +45,15 @@ def __init__(self, *args):
         # adapter and not directly with the framework.
         self.fw_adapter = FrameworkAdapter(self.framework)
         self.prometheus = PrometheusInterface(self, 'http-api')
+        self.am_rel_name = 'alertmanager'
```

Comment:
I don't believe this is something we will change in some foreseeable future, so I think we can reference an interface name directly at the line 49 without having to create an additional variable.

Reply:
agreed